game_controller.py

```diff
@@ -21,6 +22,18 @@ def __init__(self, citizens_dict: Dict[str, Card],
 
         self.game: Game
 
+    @property
```

Comment:
i'ts better to add Context properties implementation to the end of game_controller file.
And, please, separate these properties from other game_controller methods using text below:
\# =================================================================
\# Context implementation
\# =================================================================

Reply:
Ok, ready